exercises/go-1-a/.meta/example_imperative.go

```diff
@@ -0,0 +1,35 @@
+package go1a
+
+import (
+	"fmt"
+	"strconv"
+)
+
+// Describe takes an amount and returns a description
```

Comment:
Function comments need punctuation to conform to the standards, so this should end in a period.